plans/bitswap-tuning/test/transfer.go

```diff
@@ -29,11 +29,13 @@ var RootCidSubtree = &sync.Subtree{
 // Transfer data from S seeds to L leeches
 func Transfer(runenv *runtime.RunEnv) {
 	// Test Parameters
-	timeout := time.Duration(runenv.IntParamD("timeout_secs", 60)) * time.Second
-	leechCount := runenv.IntParamD("leech_count", 1)
-	passiveCount := runenv.IntParamD("passive_count", 0)
-	requestStagger := time.Duration(runenv.IntParamD("request_stagger", 0)) * time.Millisecond
-	fileSize := runenv.IntParamD("file_size", 200*1024*1024)
+	timeoutRaw, _ := runenv.IntParam("timeout_secs")
```

Comment:
Can we keep the old style of one-line value loading, to avoid polluting the local variable namespace?

Reply:
We can, and that's what I asked feedback on: should we change this function (and the others) to return only the value, or create a `Must...` function for each that returns only the value?

My suggestion would be: change the definitions to only return the value _and_  create a new function that lets us verify if the value is set!

Reply:
Decided to panic in case there are invalid parameters. A test can't run with invalid parameters anyway.